net/quagga/src/opnsense/scripts/quagga/diag-bgp.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+case "$1" in
+  bgp)
+    vtysh -d bgpd -c "show ip bgp"
+    ;;
+  summary)
+    vtysh -d bgpd -c "show ip bgp summary"
+    ;;
+  neighbor)
+    vtysh -d bgpd -c "show ip bgp neighbors $2"
+    ;;
+  neighbor-adv)
+    vtysh -d bgpd -c "show ip bgp neighbors $2 advertised-routes"
+    ;;
+  *)
+    echo "Usage: $0 bgp|summary|neighbor <ip>|neighbor-adv <ip>"
+    exit 1
+esac
+exit 0
```

Comment:
Ok, I have no idea how to do that, glad you are willing to help! 
Especially for bgp outputs the network/cisco guys are familiar with the layout of the output.

Thats's why I just wanted to through out the text in original format as a popup..